cmd/exo/cmd/config.go

```diff
@@ -86,6 +92,8 @@ secret={{.SecretKey}}
 			}
 			outPut = file
 			defer file.Close()
+		} else if err == nil {
+			return "", fmt.Errorf("File %q already exist", filepath)
```

Comment:
As a user, I'll have spend time in the third three getConfig to have it all thrown to the trash. What about creating the folder, checking the existence of the file before the `getConfig`?